casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -101,7 +101,7 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log] private[rholang]
   ): F[EvaluateResult] =
     for {
       _      <- runtime.deployParametersRef.set(ProtoUtil.getRholangDeployParams(deploy))
-      result <- doInj(deploy, reducer, runtime.errorLog)(runtime.cost)
+      result <- RuntimeManager.doInj(deploy, reducer, runtime.errorLog)(Sync[F], runtime.cost)
```

Comment:
`Sync[F]` s here temporarily or here to stay?

Reply:
I didn't add it, just passed it explicitly. IOW - I assume 'yes'